react/features/notifications/components/ParticipantNotification.web.js

```diff
@@ -0,0 +1,100 @@
+import Flag from '@atlaskit/flag';
+import EditorInfoIcon from '@atlaskit/icon/glyph/editor/info';
+import React, { Component } from 'react';
+
+import { translate } from '../../base/i18n';
+
+/**
+ * Implements a React {@link Component} to display a notification specifically
+ * about a partipant update.
+ *
+ * @extends Component
+ */
+class ParticipantNotification extends Component {
+    /**
+     * {@code ParticipantNotification} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The translation key to display as the title of the notification if
+         * no title is provided.
+         */
+        defaultTitleKey: React.PropTypes.string,
+
+        /**
+         * The translation arguments that may be necessary for the description.
+         */
+        descriptionArguments: React.PropTypes.object,
+
+        /**
+         * The translation key to use as the body of the notification.
+         */
+        descriptionKey: React.PropTypes.string,
+
+        /**
+         * Whether or not the dismiss button should be displayed. This is passed
+         * in by {@code FlagGroup}.
+         */
+        isDismissAllowed: React.PropTypes.bool,
+
+        /**
+         * Callback invoked when the user clicks to dismiss the notification.
+         * this is passed in by {@code FlagGroup}.
+         */
+        onDismissed: React.PropTypes.func,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func,
+
+        /**
+         * The text to display at the top of the notification.
```

Comment:
We may want to add to the docs of this one that titleKey will be used if this isn't provided, as we described for titleKey.